src/main/kotlin/com/openlattice/chronicle/sensorkit/IOSSensorApi.kt

```diff
@@ -0,0 +1,49 @@
+package com.openlattice.chronicle.sensorkit
+
+import retrofit2.http.Body
+import retrofit2.http.POST
+import retrofit2.http.Path
+import java.util.*
+
+/**
+ * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
+ */
+interface IOSSensorApi {
```

Comment:
Do we need a whole separate API for this? Seems like it could just be an overload of the upload API on whatever interface that lives on for v3.

Reply:
Moved